blocks/table/table.js

```diff
@@ -0,0 +1,11 @@
+export default function decorate(block) {
+  block.querySelectorAll('table tbody tr').forEach((row) => {
+    const cell = row.querySelectorAll('td')[1];
+    const wrapper = document.createElement('p');
+    while (cell.firstChild) {
```

Comment:
why do you wrap with `p` ?

Reply:
if I don't, the description will be vertically centered, instead of top aligned

Reply:
once the `pre` is removed from the first cell (see https://github.com/adobe/helix-gd) then it will also have no margin. so I don't think you need it:

<img width="499" alt="image" src="https://user-images.githubusercontent.com/917628/201314416-a1061b4b-5ec5-4af1-9056-4861cc9f2d59.png">

anyways, if you want to add `<p>` where is missing, I would add it to all cells. but check first, if there is not already a `<p>`. (i.e. if a cell contains more than 1 paragraph, it will add `<p>`s.

Reply:
🚀